swayidle/swayidle.1.txt

```diff
@@ -0,0 +1,59 @@
+/////
+vim:set ts=4 sw=4 tw=82 noet:
+/////
+:quotes.~:
+
+swayidle (1)
+============
+
+Name
+----
+swayidle - Screen power manager for sway
+
+Synopsis
+--------
+'swayidle' [options] [timeout <seconds> exec <parameter>
+			[resume exec <parameter>]] ...
+
+Swayidle handles idle events from sway: it sets up the lockscreen and turns
+off the screen. It is also possible to execute arbitrary commands or send
+commands to sway, by passing a number of commands at the end of the
+commandline. Swayidle will do nothing by default, it will require some
+commands or the use of -i -l or -x parameters (se below for details).
+
+Command syntax:
+[timeout <seconds> exec <parameter> [resume exec
+<parameter>]]
+
+It is possible to pass multiple commands to setup multiple idle handlers.
+
+Command description:
+	exec - Will execute the passed parameter in a shell
+
+Options
+-------
+
+*-h*::
+	Show help message and quit.
+
+*-i <idle timeout in seconds>*::
+	Turn off the screen after the system have been idle for the specified 
+	number of seconds.
+
+*-l <lock timeout in seconds>*::
+	Lock the screen after the system have been idle for the specified 
+	number of seconds.
+
+*-x <lock command>*::
+	Raise the lockscreen using the lock command, upon systemd sleep events.
```

Comment:
This should be documented as executed with a shell (and if it's not executed with a shell, it should be).